Update/onik_001.txt

```diff
@@ -4186,6 +4186,7 @@ void main()
 	ClearMessage();
 
 //「レナもシカトすんなー！２人がホテル街へ消えたって回覧す＠
+//「レナもシカトすんなー！２人がラブラブだって回覧す < PS3 voice actually says this＠
```

Comment:
I already came across quite a few small differences in the PS3 voice like this, making the text a little less "adult"
I've decided not to spend the time changing the text where the meaning doesn't change a great deal and have stopped making comments of the changes like this one.